### ISSUE TITLE
Lambda functions in services are not correcly translated to javascript

### ISSUE THREAD

**Issue by kklorenzotesta** [ptc-iot-sharing/ThingworxVSCodeProject#65]
When a lambda function is used inside a service, `this` is not translated to `_this`. For example from a file:
```
@ThingDefinition
class BugThing extends GenericThing {
  testLambda() {
    [0].forEach((_) => {
      logger.error(this.name);
    });
  }
}
```
The service inside the thing is generated with the following code:
```
var result = (function () {const METHOD_NAME = "testLambda";

    var _this = this;
    [0].forEach(function (_) {
        logger.error(this.name);
    });
}).apply(me, [])
```
which in this example leads to printing undefined since `this` doesn't refers correctly to the thing

**Comment by BogdanMihaiciuc** [comment]
I am aware of this but I haven’t yet figured out any good fix for it. A workaround you can use in the meantime is to alias this yourself and reference it that way.

**Comment by BogdanMihaiciuc** [comment]
I published version 2.1.5 of transformer and cli to npm, resolving this. Thank you for the fix
